src/components/SearchResults.js

```diff
@@ -9,7 +9,7 @@ export default function SearchResults(props) {
 
 	return (
 		<Popover focusOnMount={false}>
-			<ul>
+			<ul style={{ width: '250px' }}>
```

Comment:
This right here is the actual bugfix. The rest is a cleanup for #99

Reply:
@fabiankaegy Is that expected behavior for the `<Popover>` to be placed on the bottom left of the input?



![image](https://user-images.githubusercontent.com/5423135/151340232-db52cad0-1fc7-4d19-bca6-4d78c1af3107.png)

Reply:
@dinhtungdu I have not touched the anchoring of the popover for right now. This is how It was positioned before as far as I am aware. 

I don't have any more time to get this cleaned up more this week but I wanted to at least get the sizing & overlap bug fixed. 

I want to do an overhaul of the entire search component to be more visually pleasing and also more acessible. But that won't happen for this release.

Reply:
I agree, just want to make sure you know about it. This PR is good to go now!